src/NServiceBus.AcceptanceTests/Encryption/EndpointTemplates/DefaultServer.cs

```diff
@@ -42,7 +42,8 @@ public BusConfiguration GetConfiguration(RunDescriptor runDescriptor, EndpointCo
             builder.CustomConfigurationSource(configSource);
             builder.EnableInstallers();
 
-            builder.DisableFeature<TimeoutManager>();
+            // TimeoutManager is currently required by Sagas
+            builder.EnableFeature<TimeoutManager>();
```

Comment:
Makes sense, can you add that to the POA on the @DavidBoike pull so that we don't forget? (if not already done :)  )